LICENCE.md

```diff
@@ -16,40 +19,46 @@ optimize pattern matching. This is an optional feature that can be omitted when
 the library is built.
 
 
-THE BASIC LIBRARY FUNCTIONS
----------------------------
+COPYRIGHT
+---------
+
+### The basic library functions
 
-Written by:       Philip Hazel
-Email local part: Philip.Hazel
-Email domain:     gmail.com
+    Written by:       Philip Hazel
+    Email local part: Philip.Hazel
+    Email domain:     gmail.com
 
-Retired from University of Cambridge Computing Service,
-Cambridge, England.
+    Retired from University of Cambridge Computing Service,
+    Cambridge, England.
 
-Copyright (c) 1997-2024 University of Cambridge
-All rights reserved.
+    Copyright (c) 1997-???? University of Cambridge
+    Copyright (c) ????-2024 Philip Hazel
```

Comment:
@PhilipHazel Just for the record, which years were you employed by the university? Your contributions since your retirement will be copyrighted by you.

Reply:
Thanks!